dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionHelper.java

```diff
@@ -85,6 +88,7 @@ public void evictAndAddEntries(Collection collection) {
             List<Label> labels = entryDAO.getLabelByEntryId(entry.getId());
             List<String> labelStrings = labels.stream().map(Label::getValue).collect(Collectors.toList());
             entry.setLabels(labelStrings);
+            entry.setVerified(!versionDAO.findEntryVersionsWithVerifiedPlatforms(entry.getId()).isEmpty());
```

Comment:
Edge case question:

- A specific, non-verified, version is added to a collection
- Another of the entry's versions is verified.
- Won't this change the CollectionEntry's verified to true, when it was previously false?

Reply:
Yeah, that's interesting because I'm not sure what our intent is here. 
If the collection doesn't specify a version, we're advertising that a verified version exists even if it may not be the version the user lands on after clicking because it isn't the default version. 

If the collection does specify a version, what should win out?
a) the advertising angle? advertising that even if this specific version is not verified, some other version is?
b) the accuracy angle? the collection bookmarked a specific version

It's not clear to me

Reply:
A few years ago we were tossing around the idea of publishing versions, not workflows, which seems like the overall ideal solution (but a massive change and way outside the scope of this PR).

I'd lean towards accuracy for a specific version, but don't feel too strongly on it.

Reply:
I think I'll add a ticket. 
I could see the argument that the specific version you land on after clicking the link should dictate the verification status. but this seems inconsistent with the last modified date which is also not specific to a specific version
For example https://dockstore.org/organizations/bdcatalyst/collections/TOPMedworkflows

Reply:
https://github.com/dockstore/dockstore/issues/5039